source/blog/2017-02-16-hyperconverged-ovirt-with-ovn-part1.md

```diff
@@ -0,0 +1,203 @@
+```
+yum install -y epel-release
+yum install -y haveged
+systemctl start haveged
+systemctl enable haveged
+yum remove -y epel-release
+```
+If you are using EPEL please take care or removing it since it's going to make the installation of oVirt fail due to a dependency conflict.
+
+## Setup
+
+Now we can point your browser to https://demovm1.localdomain:9090/ to connect to cockpit running on your first host. Connect as root user.
+Point to Virtualization Tab and then Hosted Engine sub tab and choose there 'Hosted Engine with Gluster' as for picture 1.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture01.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture01.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture01.png")
+As the first step add the hostname of your three hosts on the *storage network*, since we have a single nic on each host I could simply use the FQDN but to make it more clear that's a different thing I’m going to use the host IP addresses as in picture 2.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture02.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture02.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture02.png")
+On the *Packages* step we can just keep everything as it is since we already installed what's needed.
+On the *Volumes* step it will propose by default to use three volumes but we are going to remove the vmstore one since we just want to build the minimal env for this demo lab so proceed as for picture 3. Please note that gdeploy will propose by default to use replica with arbiter mode to provide more space efficiency than a pure replica 3 mode.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture03.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture03.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture03.png")
+On the *Bricks* step remove the vmstore brick since we are not going to create the vmstore volume; point the device field to the storage device you are really going to use for the brick (vdb for both the volumes in my case); change the size of every brick to be sure they can fit on your device as in picture 4.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture04.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture04.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture04.png")
+Gdeploy will create the two gluster volumes and it will configure them to serve as best as possible for virtualization purposes. Wait until it successfully complete the volume definition as in picture 5.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture05.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture05.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture05.png")
+Now you can continue with hosted-engine deployment; you can accept the default proposed values almost in every question. The relevant questions are pointed out here.
```

Comment:
When you start hosted-engine deployment from gdeploy wizard, it will use the first volume in the list for hosted-engine deployment. Also it will configure the backup vol file servers. Also we have a common answer file for HC deployment, which has most of the std answers defined.